inactive_session_timeout/models/res_users.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 ACSONE - Cédric Pigeon
+# Copyright 2017 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, http, models
+from odoo.http import request, root
+
+from os import utime
+from os.path import getmtime
+from time import time
+
+DELAY_KEY = 'inactive_session_time_out_delay'
+IGNORED_PATH_KEY = 'inactive_session_time_out_ignored_url'
+
+
+class ResUsers(models.Model):
+    _inherit = 'res.users'
+
+    @api.model
+    def _check_session_validity(self):
+        if not request:
+            return
+        session = request.session
+        session_store = root.session_store
+        ipm = self.env['ir.config_parameter']
```

Comment:
You have removed the other model and it was introduced for having a cache mechanism to not query so many times the DB, so please restore it as it was.